### ISSUE TITLE
Denoised audio 0db on NSDTSEA

### ISSUE THREAD

**Issue by zfqy** [drethage/speech-denoising-wavenet#28]
The training process starting with the code according to "readme.md"  has been early terminated at epoch 34 with loss 0.034, I consider the trained model has satisfied some requirements and could deal with denoising but I got the 0dB audio which contains nothing but silence. I do not know whether it is related to the early stop but the very low loss makes it confusing.

**Comment by ffxz** [comment]
@zfqy do you deal with 0 db problem?

**Comment by zfqy** [comment]
Nope, but I discover a phenomenon that the denoised results are more likely to be silence with more epochs input into training process. Some input audio pieces are randomly set to be zero in the code with the rate 10% ( config.json : "noise_only_percent": 0.1), I haven't tried this yet but you could try and set it to zero. 


Good luck.


------------------ 原始邮件 ------------------
发件人: "FFXZ"<notifications@github.com>;
发送时间: 2019年1月2日(星期三) 晚上10:15
收件人: "drethage/speech-denoising-wavenet"<speech-denoising-wavenet@noreply.github.com>;
抄送: "若水"<649977910@qq.com>; "Mention"<mention@noreply.github.com>; 
主题: Re: [drethage/speech-denoising-wavenet] Denoised audio 0db on NSDTSEA(#28)




@zfqy do you deal with 0 db problem?

**Comment by ffxz** [comment]
@zfqy thank you, I'll try your ways. and i find that using theano backend will get true result. But when i rewrite it with tf.keras(tensorflow 1.12) ,  0db occurs.

**Comment by ffxz** [comment]
![image](https://user-images.githubusercontent.com/16372416/50622143-0d7e0200-0f46-11e9-8828-d6b91452caa8.png) I think you're True. I change noise_only_percent to zero, and then i get correct results as picture shows.

**Comment by ffxz** [comment]
![image](https://user-images.githubusercontent.com/16372416/50622200-68175e00-0f46-11e9-9ed5-1b91d9840227.png)
@zfqy  I think you're True. I change noise_only_percent to zero, and then i get correct results as picture shows.

**Comment by zfqy** [comment]
Congratulations bro. Thanks for your consideration and it helps me indeed.




------------------ 原始邮件 ------------------
发件人: "FFXZ"<notifications@github.com>;
发送时间: 2019年1月3日(星期四) 上午10:58
收件人: "drethage/speech-denoising-wavenet"<speech-denoising-wavenet@noreply.github.com>;
抄送: "若水"<649977910@qq.com>; "Mention"<mention@noreply.github.com>; 
主题: Re: [drethage/speech-denoising-wavenet] Denoised audio 0db on NSDTSEA(#28)





 @zfqy  I think you're True. I change noise_only_percent to zero, and then i get correct results as picture shows.

**Comment by NAM-hj** [comment]
Does anyone know the reason why noise augmentation shows this 0dB problem?
I tried 0.0 noise_only_augmentation setting several times. 
For the lower epochs there is sound in *_denoisedNdB.wav,
but for the higher epoch (long, over 50 epoch) then I got the denoised0dB.wav (which contains silence only.)
